packages/terra-data-grid/tests/jest/DataGrid.test.jsx

```diff
@@ -98,19 +96,19 @@ describe('DataGrid', () => {
       expect(rowComponent.props.cells).toEqual(data.cells);
     };
 
-    const wrapper = shallowWithIntl(
+    const wrapper = mountWithIntl(
       <DataGrid
         id="test-terra-data-grid"
         overflowColumns={dataFile.cols}
         rows={dataFile.rows}
       />,
-    ).dive().dive();
+    );
 
-    const rows = wrapper.find(Row);
+    const rows = wrapper.find('Row');
     expect(rows).toHaveLength(dataFile.rows.length);
-    verifyRow(0, rows.get(0), dataFile.rows[0], dataFile.cols);
-    verifyRow(1, rows.get(1), dataFile.rows[1], dataFile.cols);
-    verifyRow(2, rows.get(2), dataFile.rows[2], dataFile.cols);
+    verifyRow(0, rows.get(0), mapDataGridRow(dataFile.rows[0]), dataFile.cols.map(columns => mapDataGridColumn(columns)));
+    verifyRow(1, rows.get(1), mapDataGridRow(dataFile.rows[1]), dataFile.cols.map(columns => mapDataGridColumn(columns)));
+    verifyRow(2, rows.get(2), mapDataGridRow(dataFile.rows[2]), dataFile.cols.map(columns => mapDataGridColumn(columns)));
```

Comment:
I would not expect you to have to call the map function in your tests.  You should be testing for a known value not the result of a function.